src/roomsyncroniser.ts

```diff
@@ -51,6 +51,11 @@ export class RoomSyncroniser {
 	) {
 		this.roomStore = this.bridge.roomStore;
 		this.mxidLock = new Lock(MXID_LOOKUP_LOCK_TIMEOUT);
+		if (this.config.metrics.enabled) {
+			const rooms = roomStore.getAll();
```

Comment:
roomStore.getAll is a promise and promises can't be `await`'ed in a constructor. so, you'd need to do something like

```
roomStore.getAll().then((rooms) => {
  this.bridge.blahblahblah
});
```